modules/internal/src/main/scala/client/fs2Calls.scala

```diff
@@ -60,14 +62,14 @@ object fs2Calls {
       input: Stream[F, Req],
       descriptor: MethodDescriptor[Req, Res],
       channel: Channel,
-      options: CallOptions)(implicit S: Scheduler): Stream[F, Res] =
+      options: CallOptions)(implicit EC: ExecutionContext): Stream[F, Res] =
     monixCalls
       .bidiStreaming(
         Observable
           .fromReactivePublisher(input.toUnicastPublisher),
         descriptor,
         channel,
         options)
-      .toReactivePublisher
+      .toReactivePublisher(Scheduler(EC))
```

Comment:
what does the apply method? any penalizations on invoking it on each call?

Reply:
I think it's just a _decorator_ atop of the `ExecutionContext`, which would be the one used for executing all the "runnables".